### PR TITLE
Fixes command announcement encoding.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Also makes some inputs into tgui_inputs.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/modular_computers/file_system/programs/command/command_and_communications.dm [hunk 1]
```diff
@@ -120,13 +120,13 @@
...
 					return
-				var/input = tgui_input_text(usr, "Please write a message to announce to the station crew.", "Priority Announcement", multiline = TRUE)
+				var/input = tgui_input_text(usr, "Please write a message to announce to the station crew.", "Priority Announcement", multiline = TRUE, encode = FALSE)
 				if(!input || computer.use_check_and_message(usr))
 					return FALSE
 				var/was_hearing = HAS_TRAIT(computer, TRAIT_HEARING_SENSITIVE)
 				if(!was_hearing)
 					computer.become_hearing_sensitive()
-				usr.say(input)
+				usr.say(sanitize(input))
```
Comment (FluffyGhoster) [inline code/modules/modular_computers/file_system/programs/command/command_and_communications.dm:129]: This should probably use `strip_html_full` instead

File: code/modules/modular_computers/file_system/programs/command/command_and_communications.dm [hunk 2]
```diff
@@ -139,7 +139,7 @@
 					if(centcomm_message_cooldown)
 						to_chat(usr, SPAN_WARNING("Arrays recycling. Please stand by."))
 						return TRUE
-					var/input = sanitize(input(usr, "Please choose a message to transmit to \[ABNORMAL ROUTING CORDINATES\] via quantum entanglement.  Please be aware that this process is very expensive, and abuse will lead to... termination. Transmission does not guarantee a response. There is a 30 second delay before you may send another message, be clear, full and concise.", "To abort, send an empty message.", "") as null|text)
+					var/input = sanitize(tgui_input_text(usr, "Please choose a message to transmit to \[ABNORMAL ROUTING CORDINATES\] via quantum entanglement.", "Emergency M&#e55sage", multiline = TRUE, encode = FALSE))
```
Comment (FluffyGhoster) [inline code/modules/modular_computers/file_system/programs/command/command_and_communications.dm:142]: `M&#e55sage` ?

Comment (NonQueueingMatt) [reply code/modules/modular_computers/file_system/programs/command/command_and_communications.dm:142]: it's the emagged command announcement - meant to show that it's glitchy

**Review by FluffyGhoster — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Comment by FluffyGhoster** [comment]
Merge master

**Pull request merged**